hadoop-hdds/common/src/main/java/org/apache/hadoop/ozone/OzoneConfigKeys.java

```diff
@@ -188,6 +188,10 @@ public final class OzoneConfigKeys {
       "ozone.client.ec.grpc.write.timeout";
   public static final String OZONE_CLIENT_EC_GRPC_WRITE_TIMEOUT_DEFAULT = "30s";
 
+  public static final String OZONE_EC_GRPC_ZERO_COPY_ENABLED =
```

Comment:
Due to the complexity of the RATIS version of this maybe it makes sense to mark this as for EC? For reads it might not make any difference but for write we might want to turn one off.